tests/e2e/support/api/http.ts

```diff
@@ -10,24 +10,21 @@ export const request = async ({
   body,
   user
 }: {
-  method: 'POST' | 'DELETE' | 'PUT' | 'GET'
+  method: 'POST' | 'DELETE' | 'PUT' | 'GET' | 'MKCOL'
   path: string
   body?: BodyInit
   user?: User
 }): Promise<Response> => {
-  return await fetch(
-    join(config.backendUrl, path + (path.includes('?') ? '&' : '?') + 'format=json'),
-    {
-      method,
-      body,
-      headers: {
-        'OCS-APIREQUEST': true as any,
-        ...(user && {
-          Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
-        })
-      }
+  return await fetch(join(config.backendUrl, path + (path.includes('?') ? '&' : '?')), {
```

Comment:
I think, `format` param is required for oC10
```suggestion
  return await fetch(join(config.backendUrl, path + (path.includes('?') ? '&' : '?') + 'format=json'), {
```

Reply:
for the above request does not support query param format maybe if could do like this `join(config.backendUrl, path + (path.includes('?') ? '&' : '?') + ((!config.ocis) ? 'format=json' : ''))`? to work?

Reply:
I did same: 
```
const formatJson = config.ocis ? '' : 'format=json'
  return await fetch(join(config.backendUrl, path + (path.includes('?') ? '&' : '?') + formatJson)
```

but when I create folder in oc10 I get error: 
`FetchError: invalid json response body at http://host.docker.internal:8080/login reason: Unexpected token < in JSON at position 0`

Reply:
That might be because the MKCOL request returns an empty response body.

Reply:
@ScharfViktor may be it because you are using `dav/spaces` endpoint for `oc10`

Reply:
creating a folder in oc10 without `'format=json'` works, so I added `formatJson = false` to the `request` method.